src/api/wallet.ts

```diff
@@ -50,6 +50,15 @@ export const getAccountAddress: () => Promise<string | null> = async () => {
   return null;
 };
 
+export const getWalletNetwork: () => Promise<string> = async () => {
```

Comment:
Instead of returning a string, it should return a member of an enum of valid wallet networks (i.e. `'Devnet' | 'Testnet' | 'Localhost'`).